package.json

```
@@ -0,0 +1,71 @@
+{
+  "name": "react-surveyor",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@nimbl3/eslint-config-nimbl3": "2.1.1",
```

@gutakk Regarding the previous comment, the package name here should be changed too 😅 
```suggestion
    "@nimblehq/eslint-config-nimble": "2.1.1",
```

---

Fixed in 1719516511676185d618ff1b59033bc05765c0f9